### DISCUSSION TITLE
setting start_time in mf4 file

### DISCUSSION THREAD

**Discussion by higgx2** [danielhrisca/asammdf#684] [Q&A] [answered]
Hello,

if I create a new mf4 file from scratch the start_time is always set to 1980-01-01 00:00:00.
Is it possible to set any arbitrary date? 
I was trying with mf4.start_time = datetime(2022,4,9, 2,32,0) or mf4.__setattr__('start_time', datetime(2022,4,9, 2,32,0)) but this doesn't seem to work the way I expected.
Could anybody point out some hint, please?

Thx, Roland

**Comment by danielhrisca** [comment, marked as answer]
Almost there

```python
mf4.header.start_time = my datetime
```

**Reply by higgx2** [reply]
thx for the immediate reply and for the great work on asammdf!
